src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureColor {
+    private int firstLeg;
+    private int secondLeg;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int height, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg + secondLeg) / 2 * height;
```

Comment:
The formula used to calculate the area of an isosceles trapezoid is incorrect. It should be '((firstLeg + secondLeg) / 2) * height'. Make sure to use parentheses to ensure the correct order of operations.